src/platform/silabs/efr32/BLEManagerImpl.cpp

```diff
@@ -1153,12 +1344,31 @@ extern "C" void sl_bt_on_event(sl_bt_msg_t * evt)
     }
     break;
 
-    default:
+    default: {
         ChipLogProgress(DeviceLayer, "evt_UNKNOWN id = %08" PRIx32, SL_BT_MSG_ID(evt->header));
         break;
     }
+    }
 
+    // Unlock the stack
     chip::DeviceLayer::PlatformMgr().UnlockChipStack();
 }
 
+} // namespace Internal
+} // namespace DeviceLayer
+} // namespace chip
+
+extern "C" void zigbe_bt_on_event(volatile sl_bt_msg_t * evt);
+
+extern "C" void sl_bt_on_event(sl_bt_msg_t * evt)
```

Comment:
We might want those somewhere else in the final implementation

Reply:
I'll take note, like a matter_ble_event.cpp? I like the idea, I just want to avoid going through adding stuff for the gn builds for now as I would ideally not test gn builds since they take forever. I'll put a todo in there for now.